fenix/app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -1702,11 +1702,14 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         }
     }
 
-    val feltPrivateBrowsingEnabled: Boolean
-        get() {
+    val feltPrivateBrowsingEnabled by lazyFeatureFlagPreference(
+        key = appContext.getPreferenceKey(R.string.pref_key_should_enable_felt_privacy),
+        featureFlag = true,
```

Comment:
`Config.nightlyOrDebug` just in case?

Reply:
Is `FxNimbus.features.privateBrowsing.value().feltPrivacyEnabled` already `true`? If this is still used for experimentation and we set `featureFlag = true`, we could run into scenarios where we are setting the initial preference to `false` if they are on the `off` treatment group and that will persist unless we remove the preference.

Reply:
My bad. This is right!